calico_containers/requirements.txt

```diff
@@ -2,7 +2,7 @@ sh==1.11
 netaddr==0.7.15
 six==1.9.0
 subprocess32
-git+https://github.com/projectcalico/python-etcd.git
+git+https://github.com/jplana/python-etcd.git
```

Comment:
We don't need this; `calico` and `libcalico` have it as deps.  Now that we're not dependent on our own fork, just let pip sort it out.  We don't directly import it anyway.